setup.cfg

```diff
@@ -39,7 +39,7 @@ install_requires =
     attrs
     dataclasses ; python_version < '3.7'
     protobuf >=3.5.0.post1, < 5 # Per https://github.com/grpc/grpc/blob/v1.45.3/requirements.txt
-    msgpack
+    msgpack >=0.5.0 # 0.5.0 is needed for strict_types argument, which is required for our pickling behaviour
```

Comment:
Same change should be made in `environment_unix.yml`

Reply:
done